### PR TITLE
changed cloud.redhat.com to console.redhat.com

### PR DESCRIPTION
**Pull request by vikram-redhat**
Mass search and replace for cloud.redhat.com to console.redhat.com.

### CODE REVIEW

File: modules/nodes-descheduler-installing.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ The descheduler is not available by default. To enable the descheduler, you must
 * Cluster administrator privileges.
 * Access to the {product-title} web console.
 ifdef::openshift-origin[]
-* Ensure that you have downloaded the link:https://cloud.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in _Obtaining the installation program_ in the installation documentation for your platform.
+* Ensure that you have downloaded the link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in _Obtaining the installation program_ in the installation documentation for your platform.
```
Comment (Preeticp) [inline modules/nodes-descheduler-installing.adoc:15]: The links (previous and changed) do not take me to a site called - Red Hat OpenShift Cluster Manager site. Is this something that needs to be changed?

File: modules/understanding-telemetry-and-insights-operator-data-flow.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 The Telemeter Client collects selected time series data from the Prometheus API. The time series data is uploaded to api.openshift.com every four minutes and thirty seconds for processing.
 
-The Insights Operator gathers selected data from the Kubernetes API and the Prometheus API into an archive. The archive is uploaded to link:https://cloud.redhat.com[cloud.redhat.com] every two hours for processing. The Insights Operator also downloads the latest Insights analysis from link:https://cloud.redhat.com[cloud.redhat.com]. This is used to populate the *Insights status* pop-up that is included in the *Overview* page in the {product-title} web console.
+The Insights Operator gathers selected data from the Kubernetes API and the Prometheus API into an archive. The archive is uploaded to link:https://console.redhat.com[console.redhat.com] every two hours for processing. The Insights Operator also downloads the latest Insights analysis from link:https://console.redhat.com[console.redhat.com]. This is used to populate the *Insights status* pop-up that is included in the *Overview* page in the {product-title} web console.
```
Comment (pneedle-rh) [inline modules/understanding-telemetry-and-insights-operator-data-flow.adoc:10]: @radekvokal can you please review this review this one line? I assume that the Insights Operator uses console.redhat.com now for the archiving and downloads, but I think it is useful to double-check with you. Thanks!

**Comment by dayleparker** [comment]
Looks good to me with the new links to console.redhat.com -  officially called Red Hat Hybrid Cloud Console. 

According to the official product names spreadsheet, we shouldn't use 'OCM' as an abbreviation of OpenShift Cluster Manager. I've been sticking to this in the OpenShift Cluster Manager docs, but it would be good to make an overall decision across all docs.

Another thought - would it be a good idea to start referring to the site as its product name (now that we have an official name) - ie. 'You can view this in Red Hat Hybrid Cloud Console' instead of 'you can view this in console.redhat.com' ?

File: getting_started/deleting-your-cluster.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 To delete your {product-title} cluster:
 
-. From link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift], click
+. From link:https://console.redhat.com/openshift[console.redhat.com/openshift], click
  on the cluster you want to delete.
 
 . Click the *Actions* button, then *Delete Cluster*.
```
Comment (dayleparker) [inline getting_started/deleting-your-cluster.adoc:11]: Future content strategy question - the OpenShift Cluster Manager docs also cover deleting clusters, as it's a cluster-level task (not something that happens inside OpenShift - that's how we've been deciding the scope of the OCM docs). Would it be good to link to the OCM docs, or is it OK to have this in two places? https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html-single/managing_clusters/index#archiving-cluster_assembly_cluster-lifecycle

File: modules/cluster-telemetry.adoc [hunk 4]
```diff
@@ -67,7 +67,7 @@
 ifndef::openshift-origin[]
 = Telemetry access for {product-title}
 
-In {product-title} {product-version}, the Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://cloud.redhat.com/openshift[{cloud-redhat-com} (OCM)].
+In {product-title} {product-version}, the Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://console.redhat.com/openshift[{cloud-redhat-com} (OCM)].
```
Comment (dayleparker) [inline modules/cluster-telemetry.adoc:70]: We should edit 'OCM' to 'OpenShift Cluster Manager' - or 'Red Hat OpenShift Cluster Manager' If it's the first mention.

Also a link update for the Subscription doc -- https://access.redhat.com/documentation/en-us/subscription_central/2021/html/getting_started_with_the_subscriptions_service/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift_monitoring_stack_and_other_tools_for_red_hat_openshift_data_collection

**Comment by bobfuru** [comment]
Aside from comments already made, this LGTM.

**Review by mikemckiernan — APPROVED** [review]
/lgtm

**Pull request merged**